src/components/Profile.tsx

```diff
@@ -0,0 +1,83 @@
+import { Session } from "next-auth";
+import Image from "next/image";
+import {
+  BuildingOfficeIcon,
+  LinkIcon,
+  EnvelopeIcon,
+  MapPinIcon,
+  StarIcon,
+  UserIcon,
+} from "@heroicons/react/24/outline";
+import { getUserSession } from "@/entities/users";
+import makeImageSizes from "@/shared/makeImageSizes";
+
+type Props = {
+  session: Session | null;
+  username: string;
+};
+
+const infoList = [
+  { title: "Empresa", icon: <BuildingOfficeIcon className="h-6" /> },
+  { title: "Cidade", icon: <MapPinIcon className="h-6" /> },
+  { title: "E-mail", icon: <EnvelopeIcon className="h-6" /> },
+  { title: "Site pessoal", icon: <LinkIcon className="h-6" /> },
+  { title: "LinkedIn", icon: <LinkIcon className="h-6" /> },
+  { title: "Github", icon: <LinkIcon className="h-6" /> },
+];
+
+export default function Profile({ session, username }: Props) {
+  const { name, image } = getUserSession(session);
+
+  const imageSizes = {
+    small: "20vw",
+    medium: "20vw",
+    large: "20vw",
+  };
+
+  return (
+    <div className="p-4 w-full rounded-lg bg-podPurple">
+      <div className="flex flex-col items-center justify-center lg:justify-between lg:flex-row gap-4 items-center min-w-[270px]">
+        <div className="relative rounded-full min-h-[200px] min-w-[200px] border border-white border-8">
+          {image ? (
+            <Image
+              alt={name}
+              className="object-contain rounded-full"
+              fill
+              sizes={makeImageSizes(imageSizes)}
```

Comment:
This looks weird. Can we just use a pseudo selector?
```suggestion
              className="object-contain rounded-full w-20vw md:some-width md:some-width"
              fill
              sizes={makeImageSizes(imageSizes)}
```